experimental/dds/tree2/src/test/shared-tree/summary.bench.ts

```diff
@@ -25,7 +25,7 @@ import { makeDeepContent, makeWideContentWithEndValue } from "../scalableTestTre
 
 // number of nodes in test for wide trees
 const nodesCountWide: [numberOfNodes: number, minLength: number, maxLength: number][] = [
-	[1, 1000, 3000],
+	[1, 1000, 8500],
```

Comment:
This increase is necessary as the new schema op contains both the new schema and the previous schema. This may warrant the proposed reduction in schema index size.

Reply:
Alternatively, it would be fairly straightforward to make the schema change delta-based as the in-memory format is a btree that can efficiently compute deltas.